movie/src/components/ActionContainer/SortCard/SortCard.jsx

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardActions from "@mui/material/CardActions";
+import Collapse from "@mui/material/Collapse";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import { Box } from "@mui/system";
+import FormControl from "@mui/material/FormControl";
+import NativeSelect from "@mui/material/NativeSelect";
+import sortOption from "./SortOption.json";
+import "./SortCard.css";
+
+const ExpandMore = styled((props) => {
```

Comment:
isnt this `ExpandMore` component the same as the on in `/FilterCard.jsx` ? if so please make it reused-able